tests/fixtures/optimizer/optimizer.sql

```diff
@@ -701,3 +701,76 @@ SELECT
     "x"."a" * "x"."b" - "x"."b"
   ) AS "f"
 FROM "x" AS "x";
+
+# title: wrapped table without alias
+# execute: false
+SELECT * FROM (tbl);
```

Comment:
can you add tests that don't just do select *?

Reply:
Yep still have some testing to do for sure

Reply:
Added some more tests but want to add even more, not sure if I covered all interesting cases.